lib/getEmbeds.ts

```
@@ -24,70 +24,109 @@ export function isVideo(url?: string) {
+};
+export type FrameMetadata = {
+  version: "vNext";
+  image: string;
+  image_aspect_ratio: "1:1";
+  buttons: Array<{
+    index: number;
+    title: string;
+    action_type: "post";
+    target: string;
+  }>;
+  input: {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)